PairInteraction.jl/test/test_quantum_defect.jl

```diff
@@ -0,0 +1,21 @@
+# TestQuantumDefect
+using Test
+using SQLite
+using PairInteraction
+
+# test_comparison
+qd = PairInteraction.QuantumDefect("Rb", 78, 1, 0.5)
+
+sqldb = SQLite.DB("pairinteraction/databases/quantum_defects.db")
```

Comment:
The working directory for the tests is now the testsuite directory, so we can use a relative path for the database.